test/core/runtime/block_builder_api_test.cpp

```diff
@@ -85,6 +85,6 @@ TEST_F(BlockBuilderApiTest, InherentExtrinsics) {
  * @then the result of the check is obtained given that the provided arguments
  * were valid
  */
-TEST_F(BlockBuilderApiTest, FinalizeBlock) {
+TEST_F(BlockBuilderApiTest, DISABLED_FinalizeBlock) {
```

Comment:
This is disabled intentionally?

Reply:
Yes. This test does not make any sense. We cannot check finalize block like that